### DISCUSSION TITLE
Bows, crossbows and other non-firearm ranged weapons should have some armour-pierce.

### DISCUSSION THREAD

**Discussion by irRegularGuy646** [CleverRaven/Cataclysm-DDA#47458] [Ideas]
I've seen arrows shot from regular bows punch through plywood on youtube, slings can fire rocks with comparable force to handgun bullets, and heavy crossbows were designed to pierce through plate armour. having all ranged weapons that don't shoot bullets have 0 armour-pierce doesn't make all that much sense to me.

**Comment by actual-nh** [comment]
Discuss after 0.F is out? Currently, some (more-agreed-upon) bugfixes are needed (see #47509, for instance).

**Reply by irRegularGuy646** [reply]
just giving a suggestion, I don't really expect immediate action.

**Comment by ZhilkinSerg** [comment]
> slings can fire rocks with comparable force to handgun bullets

Since when an order of magnitude lower force is comparable?

> heavy crossbows were designed to pierce through plate armour

I am pretty sure they were designed for siege warfare - hurling stones and such at them castles.

**Reply by Zireael07** [reply]
Hurling stones and the like is ballistae, not heavy crossbows.
Heavy crossbows *were* made to punch through plate armour, and they did.
Re: slings and bullets, nope, not comparable.

**Reply by irRegularGuy646** [reply]
originally I'd heard the figure that a sling in the hands of a professional can hit with force comparable to .44 magnum randomly on the internet

I decided to do the math myself this time:
an expert can fire a sling bullet at 160 km/h (~44.448 m/s) according to [this](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjc0uLD0uzuAhVF7qwKHSQ2BWYQFjACegQIChAC&url=https%3A%2F%2Fwww.scientificamerican.com%2Farticle%2Fwhistling-sling-bullets-were-roman-troops-secret-weapon%2F&usg=AOvVaw2gkjOjZXccRMV8h_y7z2i2)

sling bullets weighed up to 2 ounces (~.056699kg) (same source as before)
the formula for force is mass in kilograms times acceleration in meters/second for a value in newtons.
.056699 x 44.448=2.520157152 newtons of force
I applied the same formula to a [cartridge of 9x19 parabellum, 115 grain Federal FMJ](https://en.wikipedia.org/wiki/9%C3%9719mm_Parabellum) and got 2.682 newtons of force

that's pretty comparable to me
(in case you're going to complain about "but that's an expert", I said *can*, and I'm pretty sure a high throwing level can be inferred as expertise with slings)

**Reply by ZhilkinSerg** [reply]
Bullet speed is an order of magnitude higher.

**Reply by Fosheze** [reply]
> Bullet speed is an order of magnitude higher.

This right here.
A regulation maximum weight bowling ball weighs 7.27Kg. If we do the math that would mean that a bowling ball moving at aprox 0.37 m/s (1.33 kph) has the same force as that .44 magnum bullet. Force isn't the only thing that goes into determining damage especially when it comes to piercing armor. There is no way a sling is going to do anything to somebody in any sort of decent armor unless it hits an unarmored spot.

**Reply by actual-nh** [reply]
A couple of things:
- 2 ounces is ~56.7 g, assuming US ounces - not .056699. (And force isn't mass * speed; that's momentum - but still a valid way of looking at the effects of a small projectile through armor.)
- The impact of a bowling ball will be spread out over a considerably larger area - this being a large part of how chainmail or kevlar can help you.
- A sling bullet is sort of like a pick-head on a polearm - heavy but concentrated in a small area, in order to get through armor.

**Reply by Fosheze** [reply]
Thank you for catching my janky math there. I knew something was off. 

Just out of curiosity I did do a bit of research and I found a couple of good summary videos (linked below) from a guy who is by no means an expert but is competent enough to use a sling effectively. This puts him roughly at the level I would expect an average survivor to be. No survivor is going to be using a sling like a professional roman slinger out of the gate; that takes literal years of practice. In the first video you can see him launch a lead slug through 1/2" OSB and leave a dent in 1/2" plywood. This is honestly better than I expected but even so it is far from armor piercing even with the lead shot. I know from personal experience that you can punch a hole in 1/2" OSB with a hammer if you miss the nail hard enough and I wouldn't consider a hammer as having an armor piercing value.

In the second video he goes over the math and with lead shot he finds that he can get roughly the same momentum out of the shot as a 9mm bullet. Even if we disregard the fact that he gets that momentum with a larger slower more poorly shaped projectile and we assume that it does actually have all the penetrating power of a 9mm bullet then that still means that it shouldn't have an armor pierce value by current game standards. If you look at the 9mm pistols in game you will see that they don't have any armor pierce value.

Arguably they would be able to get through thinner leather and more durable fabrics like denim. However they still wouldn't touch any dedicated body armor outside of the blunt damage. Honestly I thought bows were underpowered as well but now that I know that low end pistols have an AP of 0 and you can get an AP of 1 with a broadhead arrow or an AP of 3 with a bodkin arrow, bows might still be a little overpowered on the armor piercing side of things.
[Video 1](https://www.youtube.com/watch?v=FaQRNj0WpYU)
[Video 2](https://www.youtube.com/watch?v=BehBbNQRjXw)

**Reply by actual-nh** [reply]
(Actually, I would consider a hammer as having potential for armor-piercing; take a look at the business ends of quite a few polearms.)

Good point re arrowheads - and the same can be true of sling bullets. Whether you're firing off a real lead sling bullet with a () shape, a rock that you've selected and/or done some shaping on, or just any piece of rubble that you can pick up is likely to make a major difference.

**Reply by kevingranade** [reply]
You're using momentum instead of energy.  We use energy for blunt projectiles because we've come across a number of sources indicating that kinetic energy, not force (not actually a thing, but close) or momentum is what correlates with damage on blunt impact.

We use momentum for impact of *sharp* projectiles because they utilize mechanical advantage to turn momentum into penetration, but sling stones would not function like this.

Looking at kinetic energy instead of momentum, that 2oz sling stone moving at 100MPH has approximately 56J of kinetic energy.
A 9mm bullet has approximately 500J of kinetic energy, i.e. an order of magnitude higher.
This puts that sling stone at 7-8 damage based on our current formula (square root of KE in Joules), marginally higher than a .22 CB round, much lower than a .22 LR round.  Keeping in mind this is still subject to "good hit" or "critical" modifiers, this doesn't strike me as particularly far off the mark, and in fact, a rock-in-a-sling, which *should* be strictly lower than this due to much lower density, produces 7 "stab" and 4 "bullet" damage, meaning especially against unarmored targets, it greatly outperforms projections based on kinetic energy.

"armor piercing" is a red herring here, that is a feature of a projectile that allows it to better penetrate armor without increasing damage.  If a projectile merely hits harder, that's not armor pierce, that's just higher damage.
